pymongo/daemon.py

```diff
@@ -24,6 +24,8 @@
 import sys
 
 # The maximum amount of time to wait for the intermediate subprocess.
+from pymongo.errors import PyMongoError
```

Comment:
Can you move the import up so that the comment above matches with `_WAIT_TIMEOUT`?

Reply:
fixed!